### PR TITLE
New package: python-atomicwrites-0.1.8

### PR DESCRIPTION
**Pull request by okapia**
This is one of the dependencies of vdirsyncer. I don't really need the Python 3.4 variant but I guess it is good form to build both, right?

### CODE REVIEW

**Comment by pullmoll** [comment]
The template is missing
```
post_install() {
        vinstall COPYING
}
```
or whatever the MIT license file name is.

And you need to create a symlink from `python-atomicwrites` to `python3.4-atomicwrites` and add the latter to the pr, too.

You should probably install the `xtools` package and check your templates locally like
```
$ xlint srcpkgs/python-atomicwrites/template
```
to see what's missing or wrong.

**Comment by okapia** [comment]
Thanks. I've updated the pr including the symlink and license part.

File: srcpkgs/python-atomicwrites/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'python-atomicwrites'
+pkgname=python-atomicwrites
+version=0.1.8
+revision=1
+noarch=yes
+wrksrc="atomicwrites-${version}"
+build_style="python-module"
+python_versions="2.7 3.4"
+hostmakedepends="python-setuptools python3.4-setuptools"
+makedepends="${hostmakedepends}"
```
Comment (asergi) [inline srcpkgs/python-atomicwrites/template]: `makedepends` is unnecessary

File: srcpkgs/python-atomicwrites/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+depends="python"
+pycompile_module="atomicwrites"
+short_desc="Simple Python2 API for atomic file writes"
+maintainer="Oliver Kiddle <okiddle@yahoo.co.uk>"
+license="MIT"
+homepage="https://pypi.python.org/pypi/atomicwrites/0.1.8"
```
Comment (asergi) [inline srcpkgs/python-atomicwrites/template]: homepage="https://github.com/untitaker/python-atomicwrites"

File: srcpkgs/python-atomicwrites/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	vlicense LICENSE
+}
+
+python3.4-atomicwrites_package() {
+	noarch=yes
+	pycompile_version="3.4"
+	pycompile_module="atomicwrites"
+	short_desc="${short_desc/Python2/Python3.4}"
+	pkg_install() {
+		vmove usr/lib/python3.4
```
Comment (asergi) [inline srcpkgs/python-atomicwrites/template]: vlicense LICENSE

**Comment by okapia** [comment]
Thanks. I've corrected those points.

**Pull request merged**